src/engine/init.c

```diff
@@ -636,9 +637,10 @@ server_init(int argc, char *argv[])
 
 	/* initialize the network layer */
 	ctx_nr = dss_ctx_nr_get();
+	crt_swim_idx = dss_ctx_get_swim_ctx();
```

Comment:
Not sure why a function is used here, I see '1' is directly used in all other places.

Reply:
It will return 1. Hardcoded value is stay in tests mostly.